src/modules/launcher/Plugins/Microsoft.Plugin.Indexer/Properties/Resources.resx

```diff
@@ -154,7 +154,7 @@
     <value>Returns files and folders</value>
```

Comment:
@arjunbalgovind .  How will this behave one the loc team translates this?

Reply:
As per Alekhya's comment https://github.com/microsoft/PowerToys/pull/6547#issuecomment-690598915, these strings aren't used for the Indexer settings logic which this PR fixes so it shouldn't cause an issue. The plugin names/descriptions aren't user facing right now (I think they were present in the resources because the names were used in the Wox Settings UI). I know that the plugin names are used by the GetTranslatedPluginName methods in each plugin, but I don't know exactly what that method is used for.